src/DICOM.jl

```diff
@@ -176,7 +176,7 @@ function pixeldata_parse(st, sz, vr, dcm)
         read!(st, data)
     else
         # start with Basic Offset Table Item
-        data = [element(st, false)]
+        data = Array{Any,1}(element(st, false))
```

Comment:
vector with type specification can also be written as `Any[...]`, depending on personal aesthetics.